ci/prow/e2e-crd-variable

```diff
@@ -28,10 +28,10 @@ echo " Add an kmm-ci Module that contains a valid mapping..."
 oc apply -f ci/module-kmm-ci-variable.yaml
 
 echo "Check that the module gets loaded on the node..."
-timeout 1m bash -c 'until check_kmm_kmod; do sleep 3; done'
+timeout 1m bash -c 'until check_kmm_mod; do sleep 3; done'
```

Comment:
I would change the function name instead `check_kmm_kmod`.
Also we can fix the indention in the function in this PR.